test/functional/combine_logs.py

```diff
@@ -11,6 +13,9 @@
 import os
 import re
 import sys
+import tempfile
+
+from test_framework.test_framework import TMPDIR_PREFIX
```

Comment:
Can we remove this dependency? I have a `combine_logs.py` installed in a separate bin directory and it currently works as a standalone util. Adding this dependency breaks that.

Perhaps just directly define `TMPDIR_PREFIX` as `"bitcoin_func_test_"` here. If you want to go wild, you could add a unit test that checks that `TMPDIR_PREFIX` is defined the same in both files.

Reply:
Perhaps also add a comment to warn people against importing from `test_framework` in here (or I can do it in a separate PR).

Reply:
Fixed